src/Apps/Comparables/Filter/MediumFilter.tsx

```diff
@@ -0,0 +1,50 @@
+import { FilterState } from "Apps/Collect/FilterState"
+import React from "react"
+
+import { Radio } from "@artsy/palette"
+import { SystemProps } from "Artsy/SystemContext"
+
+export const MediumFilter: React.SFC<{
+  filters: FilterState
+  mediator: SystemProps["mediator"]
+}> = ({ filters, mediator }) => (
+  <>
+    {Object.keys(CategoryToMediumGeneMap).map((category, index) => {
+      const medium = CategoryToMediumGeneMap[category]
+      const isSelected = filters.state.medium === medium
+
+      return (
+        <Radio
+          my={0.3}
+          selected={isSelected}
+          value={medium} // to do: url
+          onSelect={({ selected }) => {
+            if (selected) {
+              return filters.setFilter("medium", medium, mediator)
+            } else {
+              return filters.unsetFilter("medium", mediator)
+            }
+          }}
+          key={index}
+        >
+          {category}
+        </Radio>
+      )
+    })}
+  </>
+)
+
+export const CategoryToMediumGeneMap = {
```

Comment:
This is sort of weird. Basically, the 'category' of an artwork (ie- painting, sculpture, etc.) isn't something we can search for. Instead, we need to derive the medium gene that corresponds to the category, and filter by medium. Setting medium genes from category is automatic.

It's a bit roundabout!